node/src/ui_gateway/websocket_supervisor.rs

```diff
@@ -1289,6 +1303,39 @@ mod tests {
             }
         );
         assert_eq!(ui_gateway_recording.len(), 1);
+        let mail = rx.try_recv().unwrap();
+        let inner_clone = mail.lock().unwrap();
+        assert!(inner_clone.client_by_id.is_empty());
+        assert!(inner_clone.client_id_by_socket_addr.is_empty());
+        assert!(inner_clone.socket_addr_by_client_id.is_empty())
+    }
+
+    #[test]
+    fn close_connection_logs_inability_to_flush_close_message_before_the_client_is_dumped_anyway() {
```

Comment:
you could change the name slightly and also add an assertion that client and socket address are gone